src/s2s/mongoose_s2s_info.erl

```diff
@@ -0,0 +1,46 @@
+%% Some ugly code only used in tests.
```

Comment:
I think that we should either leave it in `ejabberd_s2s` or rewrite it as less ugly code, and then put it in `mongoose_s2s_info`. We don't just move code and rename to `mongooseim`.